web/character/investigation.py

```diff
@@ -1461,11 +1463,14 @@ def share_clues(self):
     def disp_clue_table(self):
         table = PrettyTable(["{wClue #{n", "{wSubject{n", "{wType{n"])
         discoveries = self.clue_discoveries.select_related('clue').order_by('date')
+        if "tagsearch" in self.switches:
+            msg = "{wMatching Clues{n\n"
+            discoveries = discoveries.filter(Q(clue__search_tags__name__iexact=self.args)).distinct()   
         if "search" in self.switches:
             msg = "{wMatching Clues{n\n"
             discoveries = discoveries.filter(Q(message__icontains=self.args) | Q(clue__desc__icontains=self.args) |
                                              Q(clue__name__icontains=self.args) |
-                                             Q(clue__search_tags__name__iexact=self.args)).distinct()
+                                             Q(clue__search_tags__name__iexact=self.args)).distinct() 

```

Comment:
Finally getting to this after I finished the python 2->3 change. Sorry for making you wait for so long! Looks good to me, though I'd like you to remove the trailing whitespace at the end of line 1473
```suggestion
                                             Q(clue__search_tags__name__iexact=self.args)).distinct()
```